data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,753 @@
+# or privacy rights.
+#
+# Unless expressly stated otherwise, the person who associated a work with this deed makes no
+# warranties about the work, and disclaims liability for all uses of the work, to the fullest
+# extent permitted by applicable law.
+#
+# When using or citing the work, you should not imply endorsement by the author or the affirmer.
+
+
+mission "You Are A Pirate"
+	landing
+	invisible
+	source
+		government "Pirate"
+	to offer
+		not "chosen sides"
+		or
+			and
+				or
+					"reputation: Republic" < -10
+					"reputation: Syndicate" < -10
+					"reputation: Free Worlds" < -10
+				"reputation: Merchant" < -10
+			"reputation: Pirate" > 20
+	on offer
+		conversation "You Are A Pirate"
+	on accept
+		event "became a pirate"
+		fail
+
+conversation "You Are A Pirate"
+	`Preparing to land is second nature by now: patch any perforations in the inner hull, get in touch with ground control, and then make sure your chosen parking pad isn't controlled by extortionists. When you disembark, it's with a pistol at your hip, bribe money in your pocket, and a swagger in your step; to leave behind any one of these is to invite trouble. Perhaps a life of violence on the high skies and association with the rougher elements of the galaxy weren't what you had in mind when you left your homeworld, but it's what you found.`
+	choice
+		`	(Piracy, smuggling, the existence of the outlaw; this is my life now.)`
+		`	(This isn't what I want. This isn't the life I imagined when I took to the skies.)`
+			decline
+	`	Resolve will carry you far in this world. Tonight it's carried you to a spaceport paint supplier; you return to your vessel with a pallet loaded with blacks and reds. Apply it to your vessel, and you will make your intentions known to the galaxy. There can be no pretense of peace when you fly girded in blood and midnight.`
+	choice
+		`	(Conceal my old life beneath bellicose black.)`
+		`	(No. The galaxy at large may consider me a pirate, but I'm not ready to adopt their colors.)`
+			decline
+	`	You don the smock and mask, turn on the compressor. Some paint covers all manner of sin. Some speaks of the intention to commit yet more. You apply the latter sort to your ship.`
+		accept
+
+
+
+mission "I Want To Be A Pirate"
+	clearance
+	invisible
+	source
+		government "Pirate"
+	destination "Greenrock"
+	to offer
+		not "You Are A Pirate: offered"
+		not "chosen sides"
+		or
+			"combat rating" > 150
+			has "ships: Interceptor"
+			has "ships: Drone"
+			has "ships: Fighter"
+			has "ships: Light Warship"
+			has "ships: Medium Warship"
+			has "ships: Heavy Warship"
+	on offer
+		conversation "I Want To Be A Pirate"
+	to complete
+		or
+			has "Pirate Intro: Boarding: done"
+			has "Pirate Intro: Wealth: done"
+			has "Pirate Intro: Domination: done"
+	on complete
+		event "became a pirate"
+		conversation "I Want To Be A Pirate Conclusion"
+
+
+
+conversation "I Want To Be A Pirate"
```

Comment:
This one is a very lengthy conversation. We generally want to avoid very long conversations. Something MZ has said is that this is a video game, not a novel. Is there anything here we could cut down, or is it necessary that the conversation have this much meat to it?